### ISSUE TITLE
Using 'er'	 and tabbing doesn't work.

### ISSUE THREAD

**Issue by farisalamin** [matthewrobertson/ERB-Sublime-Snippets#30]
This has been bothering me for a long time now, I've only just looked at it and I've found that if I change the scope to 'text.html.ruby' instead of 'text.html.rails', it starts working again.

**Comment by michaelkoper** [comment]
I've got the same issue. You got it to work @farisalamin ? 

For some reason all rails html.erb files have the scope text.html.rails instead of text.html.rails

**Comment by farisalamin** [comment]
@michaelkoper Yeah, I changed the scope to 'text.html.ruby' in the sublime-snippet file for the er shortcut and it started working fine. Just in case, you just have to find 'Open Resource' in the command palette to get to the files.

Only problem now is that all the shortcuts stopped working after I recently updated sublime.

**Comment by michaelkoper** [comment]
Thanks for replying @farisalamin ! I did the exact same thing. Open Resource and changing all the scopes of the snippets I use. It isn't ideal if something updates but for now it does the job!

Thanks!

**Comment by farisalamin** [comment]
Yeah, the original problem I had was that the er shortcut  was the only one set to `text.html.rails` and was not working so I had to switch it to `text.html.ruby`. However since the recent sublime update, most of the shortcuts stopped working until I changed them back to `text.html.rails`.

**Comment by morgangrubb** [comment]
@farisalamin Do we have any idea why the scope changed with the update of sublime?
